src/base/utils/net.cpp

```diff
@@ -43,7 +43,8 @@ namespace Utils
 
         bool isValidSubnet(const QPair<QHostAddress, int> &subnet)
```

Comment:
This function handles only one specific case of invalid subnet so it shouldn't be in Utils. BTW, it has small piece of trivial code so you can use it inplace.

Reply:
Where should this function be moved to? This code is used in multiple places so I think it makes sense as a net util.